src/assets/scripts/client/InputController.js

```diff
@@ -406,7 +406,7 @@ export default class InputController {
 
                 break;
             case KEY_CODES.MULTIPLY:
-                this.$commandInput.val(`${currentCommandInputValue} \u2B50 `);
+                this.$commandInput.val(`${currentCommandInputValue} * `);
```

Comment:
why the change here?  

remember, the `*` is interpreted differently by operating system.  the `*` on windows is not the same as a `*` on osx, hence the unicode 😉

Reply:
`u2B50` is the ⭐ character, which we aren't using anymore, it's been looking for a literal asterisk. Since that's what we're using in other cases, this simply updates the character being inserted. If you're saying that a string in js will be handled differently on a mac, then I guess that's something we'll need to deal with, albeit separately from here.

Reply:
ahh, ok, I'll allow it 😉